python-challenges/email-scheduler/devashmit/scheduler.py

```
@@ -0,0 +1,78 @@
+        return
+    msg = MIMEMultipart()
+    msg["Subject"] = subject
+    msg["From"] = f"{FROM_NAME} <{SMTP_USER}>"
+    msg["To"] = to
+    msg.attach(MIMEText(body, "plain"))
+    try:
+        with smtplib.SMTP(SMTP_HOST, SMTP_PORT) as s:
+            s.ehlo(); s.starttls(); s.login(SMTP_USER, SMTP_PASS)
+            s.sendmail(SMTP_USER, to, msg.as_string())
+        print(f"  ✅ Sent → {to}"); logging.info(f"SENT | to={to}")
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)